static/app/views/settings/account/accountIdentities.spec.tsx

```diff
@@ -49,19 +40,41 @@ describe('AccountIdentities', function () {
           status: 'can_disconnect',
           organization: null,
         },
+        {
+          category: 'org-identity',
+          id: '2',
+          provider: {
+            key: 'google',
+            name: 'Google',
+          },
+          status: 'needed_for_global_auth',
+          organization: null,
+        },
       ],
     });
 
-    render(
-      <AccountIdentities
-        route={router.routes[0]}
-        routeParams={router.params}
-        location={router.location}
-        params={router.params}
-        router={router}
-        routes={router.routes}
-      />
-    );
+    render(<AccountIdentities />);
+
+    expect(await screen.findByTestId('loading-indicator')).toBeInTheDocument();
+
+    await waitForElementToBeRemoved(() => screen.getByTestId('loading-indicator'));
```

Comment:
i've found this to be flaky since depending on how fast things are running it can error on waitForElementToBeRemoved. It expects the element to exist and then disappear and sometimes it has already disappeared.

Recommend just letting the findByText on the next line wait for it to finish loading